recipes/opusfile/all/conanfile.py

```diff
@@ -116,14 +116,16 @@ def build(self):
 
             #==============================
             # TODO: to remove once https://github.com/conan-io/conan/pull/12817 available in conan client
+            # Arm64 builds would have been updated by a patch as they need v17 of MSVC
+            expected_msvc_version = "v143" if self.version == "0.12" and self.settings.arch == "armv8" else "v140"
```

Comment:
Minor point: the patch is applied to both 0.11 and 0.12, but v143 is only selected for 0.12.